java/src/Ice/src/main/java/com/zeroc/IceInternal/ProxyFactory.java

```diff
@@ -7,50 +7,20 @@
 import com.zeroc.Ice.OperationInterruptedException;
 
 public final class ProxyFactory {
-  public com.zeroc.Ice.ObjectPrx stringToProxy(String str) {
```

Comment:
I assume you plan to remove ProxyFactory entirely in a follow-up PR?

Reply:
Yes, this PR is just about finalizing the changes to `createProxy`.
I plan to resync Java with C++ and C# in my next PR after this one.